This patch is needed because Jonas's rust-lazy-regex doesn't have the
full set of provides that rust team patches do.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -159,3 +159,3 @@
 [dependencies.lazy-regex]
-version = ">=2"
+version = "3"
 
